extensions/functions_aggregate_generic.yaml

```diff
@@ -40,3 +40,13 @@ aggregate_functions:
         decomposable: MANY
         intermediate: any1?
         return: any1?
+  - name: "first"
+    description: First of a set of values.
```

Comment:
Additional caveat:  Assumes the input is ordered.  If the input is not ordered then the result is undefined.